Formula/povray.rb

```diff
@@ -16,46 +15,38 @@ class Povray < Formula
   depends_on :macos => :lion
   depends_on "autoconf" => :build
   depends_on "automake" => :build
+  # Reported upstream 2017-09-02
+  # https://github.com/POV-Ray/povray/issues/317
+  depends_on "boost@1.60"
   depends_on "libpng"
-  depends_on "boost"
   depends_on "jpeg"
   depends_on "libtiff"
   depends_on "openexr" => :optional
 
   # Patches lseek64 => lseek
-  patch :p0 do
-    url "https://raw.githubusercontent.com/Homebrew/formula-patches/3316200/povray/patch-lseek64.diff"
-    sha256 "6ade943b074f25d35d49a82f920d81a48be7d18c9b5a6db9988020a1f9b0bda4"
-  end
-
-  # Fixes configure script's stat usage, automake subdir
-  patch :p0 do
-    url "https://raw.githubusercontent.com/Homebrew/formula-patches/3316200/povray/patch-unix-configure.ac.diff"
-    sha256 "025bfc178a6a298052ff97c0b2659b7f283d57bb0e43ba9962d3a5376ad16099"
+  patch do
+    url "https://raw.githubusercontent.com/Homebrew/formula-patches/ffb64e18221/povray/patch-lseek64.diff"
+    sha256 "0995431ffd56763fd821d3e65f53c2d799ddda0dca8859513e162e12f780cde0"
   end
 
-  # prebuild.sh doesn't create Makefile.in properly
-  patch :p0 do
-    url "https://raw.githubusercontent.com/Homebrew/formula-patches/3316200/povray/patch-unix-prebuild.sh.diff"
-    sha256 "1322acc324327019a31a2b97a59dbcfc6ca24981a833627399ae2ebd92c24ef6"
+  # Fixes configure script's stat usage
+  patch do
+    url "https://raw.githubusercontent.com/Homebrew/formula-patches/b444c92a9e1/povray/patch-unix-configure.ac.diff"
```

Comment:
Are you sure this is still needed? They changed to the current `stat --format=%u ${HOME} || stat -f %u ${HOME}` after I reported the issue. (It emits a warning to stderr, but works fine.)

Reply:
I figured I might as well just remove the warning as well, but I suppose one less patch is better.